### DISCUSSION TITLE
[Time Series issue] log transformation and finalize model issue

### DISCUSSION THREAD

**Discussion by Diogo-loureiro** [pycaret/pycaret#3445] [Q&A]
Hi Guys, 
Thanks a lot for this great lib. I am facing two issues on time series module. I have a time series (which is a white noise) but I wanted to test applying log transformation but got the following error (screen attached). I already checked and there aren't any nan values. I also attached the dataset.
[test.csv](https://github.com/pycaret/pycaret/files/11105858/test.csv)
![image](https://user-images.githubusercontent.com/86968410/228677326-723a592e-cd1d-4c10-9faf-75b30bd7ed5f.png)
Without doing any transformation, I also had an issue to finalize the model (grand means). Did a bit of researching and tested a few things but no luck at all.
![image](https://user-images.githubusercontent.com/86968410/228677969-f9fe6a1d-7a19-402d-8d83-6b93991fdda7.png)

**Comment by ngupta23** [comment]
Please open an issue and provide all the required information (code + data + version etc) to allow us to debug. Without that, it is very difficult to provide an answer by just looking at the image above. Thanks!

**Comment by ngupta23** [comment]
One thing I can say from your test.csv is that you have 0 values in there and log of 0 will produce large magnitude close to infinity. So that could be causing some of your problems.

I can not think of anything else causing the issue for the second problem that you mentioned.

**Reply by Diogo-loureiro** [reply]
@ngupta23 Thanks a lot. Those 0 explain the log issue (I will need to handle using a different approach). I will open the issue for the second issue.

**Reply by ngupta23** [reply]
Thanks. Appreciate it.

**Reply by Diogo-loureiro** [reply]
@ngupta23 I found the root cause of the issue. It was related to one of the packages. My issue has been resolved reinstalling PyCaret. Thanks for all.
